data.sql

```diff
@@ -3,4 +3,13 @@
 INSERT INTO animals(date_of_birth, escape_attempts, neutered, weight_kg, name) VALUES ('2020-03-02', 0, true, 10.23, 'Agumon');
 INSERT INTO animals(date_of_birth, escape_attempts, neutered, weight_kg, name) VALUES ('2018-11-15', 2, true, 8, 'Gabumon');
 INSERT INTO animals(date_of_birth, escape_attempts, neutered, weight_kg, name) VALUES ('2021-01-07', 1, false, 15.04, 'Pikachu');
-INSERT INTO animals(date_of_birth, escape_attempts, neutered, weight_kg, name) VALUES ('2017-05-12', 5, true, 11, 'Devimon');
\ No newline at end of file
+INSERT INTO animals(date_of_birth, escape_attempts, neutered, weight_kg, name) VALUES ('2017-05-12', 5, true, 11, 'Devimon');
+
+-- Add more data to animals table
+INSERT INTO animals(date_of_birth, escape_attempts, neutered, weight_kg, name) VALUES ('2020-02-08', 0, false,-11, 'Charmander');
+INSERT INTO animals(date_of_birth, escape_attempts, neutered, weight_kg, name) VALUES ('2020-11-15', 2, true,-5.7, 'Plantmon');
```

Comment:
- [ ] Dear @mirwaisfarahi, Please notice the year for `'Plantmon'` is ``2021/11/15`` instead of `'2020-11-15'`, This will become a problem later when you try to manipulate data, Please make the appropriate changes.

Reply:
Dear @TadesseDev, Thank you for the feedback. It has been resolved.